### DISCUSSION TITLE
will EaglerReborn beta 1.1 and plugin api 5.0 ever come out?

### DISCUSSION THREAD

**Discussion by radmanplays** [EaglerReborn/support#1] [General]
i was wondering now that eagler reborn has been dmca'd will EaglerReborn beta 1.1 and plugin api 5.0 ever come out?
also why did eagler reborn get dmca'd even though it didnt have any copyrighted content? isnt false dmca illegal?

**Comment by radmanplays** [comment]
(for otterdev if discord is blocked for him) : i uploaded eagler reborn to eaglercraft archive but i dont want to post the link here. if eagler reborn had more updates can you upload it to mediafire or something so i can put it in eaglercraft archive so it will be public? i dont care if eaglercraft archive gets dmca'd

**Reply by radmanplays** [reply]
also check your email. i invited you to eaglercraft archive

**Comment by ZXMushroom63** [comment]
very probably. I'm currently finishing off a5.0 of the plugin api.
As for how the project was dmca'd, pretty much it had the mimecraft textures and assets in the repo, which are copyrighted.

Here is an example from the dmca request:
VA 1-886-003 - diamond ore - https://publicrecords.copyright.gov/detailed-record/26472666
The diamond ore was definetly in the repo.

**Reply by radmanplays** [reply]
can you or otterdev make the compiler extract all of Minecraft textures from the .jar file when compiling and get the repo back up?

**Reply by ZXMushroom63** [reply]
thats what he's working on rn. (otter)

**Reply by wxnnvs** [reply]
Nice

**Comment by leahanderson1** [comment]
heres a preview of what you can do with plugin api a5:
![image](https://github.com/EaglerReborn/support/assets/144633084/6830da42-40a6-4cee-9255-3d7727feb264)

**Comment by ZXMushroom63** [comment]
It's back up:
https://github.com/EaglerReborn/reborn
